app/assets/stylesheets/semantic-ui/elements/_button.scss

```diff
@@ -789,41 +782,40 @@
 .ui.basic.inverted.buttons .button,
 .ui.basic.inverted.button {
   background-color: transparent !important;
-  color: #F9FAFB !important;
-  box-shadow: 0px 0px 0px 2px rgba(255, 255, 255, 0.5) inset !important;
+  color: rgba(0,0,0,.7) !important;
```

Comment:
Again use varaiables for colors, it makes it so much easier to understand the intent.